Source/DataSources/DataSourceDisplay.js

```diff
@@ -90,7 +95,7 @@ define([
     DataSourceDisplay.defaultVisualizersCallback = function(scene, entityCluster, dataSource) {
         var entities = dataSource.entities;
         return [new BillboardVisualizer(entityCluster, entities),
-                new GeometryVisualizer(scene, entities),
+                new GeometryVisualizer(scene, dataSource),
```

Comment:
This is another breaking change, correct? (I'm okay with it, just making sure we document it).

Reply:
Actually,why not just pass the primitives and ground primitive collections instead of the dataSource?  This keeps it loosely coupled.

Reply:
I was trying to cut down on the number of params, but I'm okay passing `entites, primitives, groundPrimitives`